loader/test/vts.geo.test.js

```diff
@@ -0,0 +1,70 @@
+const nock = require("nock");
+const { checkAvailability } = require("../src/sources/vts/geo");
+
+const apiResponse = require("./fixtures/vts.geo.test.json");
+const nopHandler = () => {};
```

Comment:
`noopHandler` ?

Reply:
Sure, if you like 👍🏽

Reply:
I guess I've seen `noop` for "no-op" reasonably often (don't love it), but have never seen `nop` before. 🤷

Reply:
(I usually try to use complete words, so tend to use a name like `doNothing` for no-op functions.)